scripts/build

```diff
@@ -11,31 +11,34 @@ cd "$SCRIPTS_DIR/.."
 #########
 
 function title() {
-    echo "Build Internet Identity"
+    echo "Builds Internet Identity and Archive Canisters"
 }
 
 function usage() {
     cat << EOF
 
 Usage:
-  $0 [--only-dependencies]
+  $0 [--only-dependencies] [CANISTER...]
```

Comment:
nit: CLI flags like `--internet-identity` or `--archive` are usually spelled out; `CANISTER` would be a argument that is free (i.e. read as a string) (also I've rarely seen underscores like `--internet_identity` (while it's more consistent with the internal naming of our canister, it's less consistent with CLI conventions 🙃 ))